src/services/notifications.service.js

```diff
@@ -406,11 +413,18 @@ const initialiseNotificationsStatusIfNotSet = async (fsaId, emailsToSend) => {
 const generateEmailsToSend = (registration, lcContactConfig, configData) => {
   const cy = registration.submission_language === "cy";
   let emailsToSend = [];
+  const fboEmailAddress =
+    registration.establishment.operator.operator_email ||
+    registration.establishment.operator.contact_representative_email;
 
   if (
-    registration.status &&
-    registration.status.notifications &&
-    registration.status.notifications.length > 0
+    (registration.status &&
```

Comment:
I feel like we're building on sub-optimal code here and the manual test needs to be very thorough.